src/index.ts

```diff
@@ -1,3 +1,4 @@
+export * from './messages/embed/Embed';
```

Comment:
where's the assertions export 😇

Reply:
Should we export the assertions? I thought they were supposed to be private.

Reply:
Nothing private about them and maybe they can be useful for users.. plus the application commands one are exported